layouts/shortcodes/delivery-team.html

```diff
@@ -0,0 +1,32 @@
+{{ $data := index site.Data site.Language.Lang }}
+
+{{ if $data.delivery_team.delivery_team.enable }}
+{{ with $data.delivery_team.delivery_team }}
+<section class="max-w-7xl mx-auto my-12 grid md:grid-cols-2 items-center gap-20">
```

Comment:
I'll create a follow on story to complete the styling since the design could still change.